### DISCUSSION TITLE
Running Imposter Server on an actual Webserver

### DISCUSSION THREAD

**Discussion by RedLightning42** [Impostor/Impostor#336] [Q&A]
I know Imposter is designed to be run on your local desktop and shared out to friends, but is there a way to set this on a permanent Webserver with a Static IP so my friends can play when I'm not around? I have access to a Server for normal websites, but I'm struggling to find a way to install Imposter in a permanent mode there...

(Bonus Points if anyone knows how to get Crowded Mod working on there too...)

**Comment by le-show-de-didoub74** [comment]
Hello, You can Use a VPS (virtual private server) where you configure you're server 

Here your friends can play when you are not present 
You have a private IP
And It's very simple to configure juste follow the instructions

**Comment by le-show-de-didoub74** [comment]
And for Crowded Mod, I don't know where we have to put BepInEx

**Comment by Fluffy728** [comment]
You can use a vpn
